spec/system/candidate_interface/candidate_entering_contact_details_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.feature 'Entering their contact details' do
+  include CandidateHelper
+
+  scenario 'Candidate submits their contact details' do
+    given_i_am_not_signed_in
+    and_i_visit_the_contact_details_page
+    then_i_should_see_the_homepage
+
+    given_i_am_signed_in
+    and_i_visit_the_site
+
+    when_i_click_on_contact_details
+    # and_i_incorrectly_fill_in_my_phone_number
+    # and_i_submit_the_form
+    # then_i_should_see_validation_errors
```

Comment:
if we do not need these we should remove the comments and the relevant methods.

Reply:
I think these are stubs for the following PRs which will uncomment them; this is the pattern we used for personal details and I liked it as it means we could just check the system spec to see "oh, we still need to do this because there still are comments"